Frontend/src/pages/registration_administration/components/RegistrationActions.jsx

```diff
@@ -87,20 +103,45 @@ export default function RegistrationActions({ selected, refresh }) {
           <Button
             onClick={() =>
               changeStatus(
-                [...selected.accepted, ...selected.cancelled],
+                [
+                  ...selected.accepted,
+                  ...selected.cancelled,
+                  ...selected.waiting,
+                ],
                 'pending'
               )
             }
           >
             <UiIcon name="times" /> Reject
           </Button>
         )}
-        {anyDeletable && (
+        {anyWaitlistable && (
+          <Button
+            basic
+            onClick={() =>
+              changeStatus(
+                [
+                  ...selected.pending,
+                  ...selected.cancelled,
+                  ...selected.accepted,
+                ],
+                'waiting_list'
+              )
+            }
+          >
+            <UiIcon name="hourglass" /> Delete
+          </Button>
+        )}
+        {anyCancellable && (
           <Button
             negative
             onClick={() =>
               changeStatus(
-                [...selected.pending, ...selected.cancelled],
+                [
+                  ...selected.pending,
+                  ...selected.accepted,
+                  ...selected.waiting,
+                ],
                 'cancelled'
               )
             }
```

Comment:
It won't let me comment 2 lines below, but the "Delete" should probably be "Cancel"?